packages/appstore/src/components/containers/trading-app-card-actions.tsx

```diff
@@ -46,6 +47,7 @@ const TradingAppCardActions = ({
                     onAction={onAction}
                     is_buttons_disabled={is_buttons_disabled}
                     is_real={is_real}
+                    as_disabled_deposit_button={as_disabled}
```

Comment:
``````suggestion
                    a<MultiActionButtonGroup
                    link_to={link_to}
                    onAction={onAction}
                    is_buttons_disabled={is_buttons_disabled}
                    is_real={is_real}
                    is_deposit_button_disabled={is_disabled}
```

Reply:
@mahdiyeh-deriv Answered in Slack